docs/development/design/fsm.md

```diff
@@ -28,7 +28,6 @@ stateDiagram-v2
     REGISTERED --> UNAUTHENTICATED: "logout"
     UNREGISTERED --> REGISTERING: "register_device"
     UNREGISTERED --> UNAUTHENTICATED: "logout"
-    REGISTERED --> UNAUTHENTICATED: "logout"
```

Comment:
Why is this line removed?

Reply:
duplicate: see line 28

Reply:
generated graph, shows duplicate lines from REGISTERED-> UNAUTHENTICATED (https://github.com/dave-tucker/apex/blob/b39acf40d7c51cdfdb50db6759aa1cf09151b36c/docs/development/design/fsm.md)